pkg/helmraiser/helm.go

```diff
@@ -0,0 +1,152 @@
+package helmraiser
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	jsonnet "github.com/google/go-jsonnet"
+	"github.com/google/go-jsonnet/ast"
+	"github.com/pkg/errors"
+	yaml "gopkg.in/yaml.v3"
+)
+
+func confToArgs(conf map[string]interface{}) ([]string, []string, error) {
+	var args []string
+	var tempFiles []string
```

Comment:
You only ever add one file this this, right? No need for a slice then

Reply:
I'm optimizing for future cases where we might want to pass multiple files (it is possible to have multiple --values with `helm template`) and can easily extend this without having to change this elsewhere. If you don't see value in this, I'll remove it.